packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -44,10 +234,27 @@ export interface HttpGatewayRouteSpecOptions {
    * The VirtualService this GatewayRoute directs traffic to
    */
   readonly routeTarget: IVirtualService;
+
+  /**
+   * The default target host name to write to.
+   * Boolean is used to select either true = enable or false = disable.
+   * When enabled, rewrites the original request received at the Virtual Gateway to the destination Virtual Service name.
+   * When disabled, retains the original hostname/prefix from the request.
+   *
+   * @default true
+   */
+  readonly rewriteRequestHostname?: boolean;
+
+  /**
+   * The path to rewrite.
+   *
+   * @default - rewritten to the target Virtual Service's hostname and the matched prefix is rewritten to '/'.
```

Comment:
This description suggests to me even more that this should be part of `match`, not a separate property.

Reply:
Well noted.